Makefile

```diff
@@ -33,10 +33,16 @@ build-development: build-api
 # Build PDF
 #
 check-images: # scan through markdown to validate Github flavoured markdown
-	node ./scripts/linkchecker.js ./pages/1.7
+	node ./scripts/linkchecker.js ./pages
+
+check-markdown: #scan through all the markdown and show the warnings
+	node ./scripts/mdlinter.js ./pages
+
+check-nonutf: #scan through the all the markdown and find non-utf8
+	./scripts/charcheck.sh ./pages
 
 build-pdf-production:
-	./scripts/mdtopdf.sh ./pages ./build-pdf
+	./scripts/mdtopdf.sh ./pages/1.11 ./build-pdf
```

Comment:
just 1.11? Is this first just for testing? Might it be a good idea to output at least a warning here so it should not get merged into staging and production without improvement?